python/ccf/ledger.py

```diff
@@ -232,21 +232,46 @@ def __next__(self):
             raise StopIteration() from e
 
 
+class Ledgerchunk:
```

Comment:
Could you also update `doc/operators/ledger_python.rst` as documentation is automatically created from these classes?

Reply:
Sure